content/en/roadmap.md

```diff
@@ -28,158 +28,95 @@ and security issues over new features. Depending on the volume of incoming issue
 
-- **Terraform provider**
-  - [x] [Add support for air-gapped bootstrap](https://github.com/fluxcd/terraform-provider-flux/pull/664)
-  - [x] [Implement drift detection and correction for cluster state](https://github.com/fluxcd/terraform-provider-flux/pull/661)
-  - [x] [Improve documentation with concrete examples for the various bootstrap methods](https://github.com/fluxcd/terraform-provider-flux/tree/main/examples)
+- **Troubleshoot helpers**
+  - [x] [Implement `flux debug helmrelease --show-values`](https://github.com/fluxcd/flux2/pull/5106)
+  - [x] [Implement `flux debug kustomization --show-vars`](https://github.com/fluxcd/flux2/pull/5117)
 
 - **Conformance testing**
-  - [x] [End-to-end testing for Kubernetes 1.30](https://github.com/fluxcd/flux2/pull/4734)
-  - [x] [End-to-end testing for OpenShift](https://github.com/fluxcd/flux2/issues/4625)
-  - [x] [End-to-end testing for k3s](https://github.com/fluxcd/flux2/pull/4777)
+  - [x] [End-to-end testing for Kubernetes 1.32](https://github.com/fluxcd/flux2/pull/5111)
+  - [x] [End-to-end testing for GitHub App auth](https://github.com/fluxcd/pkg/pull/818)
 
 - **EOL and Deprecations**
-  - End support for Flux v2.0.x
-  - End support for Kubernetes v1.27.x
-  - Deprecate APIs in group `helm.toolkit.fluxcd.io/v2beta2`
-  - Deprecate APIs in group `image.toolkit.fluxcd.io/v1beta1`
-  - Remove deprecated Terraform providers `flux_install` and `flux_sync`
+  - End support for Flux v2.2.x
+  - End support for Kubernetes v1.29.x
+  - Remove deprecated APIs in group `kustomize.toolkit.fluxcd.io/v1beta1`
+  - Remove deprecated APIs in group `source.toolkit.fluxcd.io/v1beta1`
+  - Remove deprecated APIs in group `notification.toolkit.fluxcd.io/v1beta1`
+  - Remove deprecated APIs in group `helm.toolkit.fluxcd.io/v2beta1`
+  - Remove deprecated APIs in group `image.toolkit.fluxcd.io/v1beta1`
 
-### v2.4 (Q3 2024)
+### v2.6 (Q2 2025)
 
-**Status**: Completed (v2.4.0 [changelog](https://github.com/fluxcd/flux2/releases/tag/v2.4.0))
+**Status**: Provisional
 
-The primary goal of this milestone is to make a generally available release for the Flux S3-compatible storage APIs.
+The primary goal of this milestone is to make a generally available release for the Flux image automation APIs.
 
-- **S3-compatible storage integrations**
-  - [x] Promote the `Bucket` API to `v1`
-  - [x] [Allow specifying a custom CA certificate in Bucket API](https://github.com/fluxcd/source-controller/issues/973)
-  - [x] [Allow specifying HTTP/S proxy in Bucket API](https://github.com/fluxcd/source-controller/issues/1493)
-  - [x] [Add support for STS endpoint in the Bucket API](https://github.com/fluxcd/source-controller/issues/1423)
+- **Image automation**
+  - [ ] Promote the `ImageUpdateAutomation` API to `v1`
+  - [ ] Promote the `ImageRepository` API to `v1`
+  - [ ] Promote the `ImagePolicy` API to `v1`
```

Comment:
Should https://github.com/fluxcd/image-reflector-controller/pull/368 and https://github.com/fluxcd/image-automation-controller/pull/508 be explicitly mentioned as part of that bump? I'm very eager to pick those up again but I would like for us to agree that we really want those to be implemented.

Reply:
I personally really want those to be implemented ❤️

Reply:
@makkes See the line below which links to the umbrella issue that you've created. That issue should track all PRs for the image digest feature.

Reply:
Ah, I missed these. Thanks!